content/200-orm/500-reference/200-prisma-cli-reference.mdx

```diff
@@ -1342,6 +1342,41 @@ Other options:
    --script > script.sql
   ```
 
+## Prisma Data Platform
+
+### <inlinecode>platform</inlinecode> ([Early Access](/platform/maturity-levels#early-access))
+
+The `platform` command provides access to the Prisma Data Platform through the Prisma CLI starting in version `5.7.0` or later.
+
+- **Authentication**:
+
+  - `platform auth login`: Opens a browser window for login or account creation.
```

Comment:
Shouldn't these all be listed in the side nav? I also wonder if repeating the `platform auth` is needed. 

![Screenshot 2024-01-30 at 1 29 26 PM](https://github.com/prisma/docs/assets/36578389/f6e2eebc-13d3-42ed-83da-77397f8b29e7)

Reply:
@mhessdev, these are bulleted list items; hence, they're not listed on the side nav. We kept it on a bulleted list for simplicity, so it's easy to maintain the information across two pages the primary information residing on the the platform CLI page. 

As an alternative, I can change it to headings and then heading view on the side nav to level 4, which looks too cluttered considering the other items on this page (I've attached two screenshots in the comment for reference).

IMO, The repeated `platform auth` helps get an idea of the complete command, similar to other pages as well. 
 
 Let me know your thoughts. 
 
<img width="506" alt="image" src="https://github.com/prisma/docs/assets/64993082/fcb0e784-c058-4d5a-98fd-6a021d65d460">
<img width="506" alt="image" src="https://github.com/prisma/docs/assets/64993082/11fe4134-5407-4c22-9398-0bfd7e1b974f">